virt/virt-4-10-release-notes.adoc

```diff
@@ -199,7 +202,14 @@ metadata:
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1977037[*BZ#1977037*])
 
-//Known issues discovered after 4.9 release
+//Known issues discovered after 4.10 release
```

Comment:
This should probably be "Known issues for 4.10 release"

Reply:
Done.